src/main.py

```diff
@@ -0,0 +1,202 @@
+from configparser import NoSectionError
+import requests
+import json
+import logging
+import os
+import re
+
+
+def setup_logger():
+    """
+    Setup logger default configuration
+    """
+    logging.basicConfig(
+        level=os.environ.get("LOGLEVEL", "INFO"),
+        format="%(asctime)s %(levelname)s %(message)s",
+    )
+
+
+def specify_github_url_name(user_name, org_name):
+    """
+    Returns a full URL with the given USER_NAME or ORG_NAME variable.
+
+    This function checks the USER_NAME or ORG_NAME variables given
+    and check conflicts
+    """
+    # Check that both variables aren´t empty at the same time
+    if user_name is None and org_name is None:
+        logging.error("USER_NAME org ORG_NAME must be filled")
+        exit(1)
+    # Check that both variables aren´t filled
+    if user_name is not None and org_name is not None:
+        logging.error(
+            "Only USER_NAME or ORG_NAME can be provided, not both at the same time"
+        )
+        exit(1)
+    if user_name is not None:
+        return "https://api.github.com/users/{}/repos".format(user_name)
+    if org_name is not NoSectionError:
+        return "https://api.github.com/orgs/{}/repos".format(org_name)
+
+
+def get_repos_topics(url, list_untagged_repos):
+    """
+    Returns a JSON with all the repositores values
+
+    Filter the repositories with their topics assigned.
+    """
+    headers = {
+        "Accept": "application/vnd.github.v3+json",
+    }
+
+    response = requests.get(url, headers=headers)
+
+    logging.debug(
+        "This is the response code from Github request: %s".format(response.status_code)
+    )
+
+    if list_untagged_repos == "true":
+        for repo in json.loads(response.content):
+            if repo.get("topics") == []:
```

Comment:
this probably is better using `if isinstance(repo.get("topics"), list) and len(repo.get("topics")) == 0` to ensure both is an array and is fullfilled, but maybe i'm extrapolating too literal from nodejs